charts/conductor/values.yaml

```diff
@@ -12,6 +12,9 @@ externalSecrets:
   parameterStore:
     name: "secret-store-parameter-store"
     kind: ClusterSecretStore
+  # remoteRef:
+  #   key: ~
+  #   secretKey: ~
```

Comment:
not needed anymore or was this for debug?

Reply:
Its commented out for reference.  We accept the values if they exist and don't if they don't.

Reply:
I will add a comment explaining.

Reply:
cool just checking!